api_tests/src/comment.spec.ts

```diff
@@ -285,11 +288,13 @@ test("Reply to a comment", async () => {
   );
   expect(replyRes.comment_view.counts.score).toBe(1);
 
-  // Make sure that comment is seen on alpha
+  // Make sure that reply comment is seen on alpha
   // TODO not sure why, but a searchComment back to alpha, for the ap_id of betas
   // comment, isn't working.
   // let searchAlpha = await searchComment(alpha, replyRes.comment);
   let postComments = await getComments(alpha, postRes.post_view.post.id);
+  // Note: in Lemmy 0.18.3 pre-release this is coming up 7
+  expect(postComments.comments.length).toBeGreaterThanOrEqual(2);
```

Comment:
Maybe other tests arent doing proper cleanup. Try running only this one.

Reply:
It just has to be 2, the test itself doesn't need to it to be a specific value.